lib/appsignal/transaction.rb

```diff
@@ -156,6 +158,29 @@ def set_tags(given_tags = {})
       @tags.merge!(given_tags)
     end
 
+    # Add breadcrumbs to the transaction.
+    #
+    # @param category [String] category of breadcrumb
+    #   e.g. "UI", "Network", "Navigation", "Console".
+    # @option action [String] name of breadcrumb
+    #   e.g "The user clicked a button", "HTTP 500 from http://blablabla.com"
+    # @option metadata [Hash<String,String>]  key/value metadata in <string, string> format
+    # @option time [Time] time of breadcrumb, should respond to `.to_i` defaults to `Time.now.utc`
+    # @return [void]
+    #
+    # @see Appsignal.add_breadcrumb
+    # @see http://docs.appsignal.com/ruby/instrumentation/breadcrumbs.html
+    #   Tagging guide
```

Comment:
```suggestion
    #   Breadcrumb reference
```

If it's a guide for the breadcrumb feature we should move it to `/guides/`